lib/flatpak-installation.c

```diff
@@ -1168,23 +1178,28 @@ progress_cb (OstreeAsyncProgress *progress, gpointer user_data)
                                   formatted_bytes_sec, formatted_bytes_transferred,
                                   formatted_total);
 
-          new_progress = (100 * bytes_transferred) / total_delta_part_size;
+
+          if (outstanding_extra_data)
+            new_progress = 10 + (35 * bytes_transferred) / total_delta_part_size;
```

Comment:
The 10 + is a bit weird here. If we have delta files I don't think we ever need to download any metadata files separatley, so make the deltas be from 0-45 instead of 10-45.